src/cargo/ops/cargo_rustc/commands.rs

```diff
@@ -0,0 +1,131 @@
+use std::collections::HashMap;
```

Comment:
I think that `engine` may also be a good name for this module to emphasize that it's the engine through which all processes are run.